packages/core/src/enable-if.ts

```diff
@@ -16,7 +16,7 @@ export function isPluginEnabled(t: PluginOrDisabledPlugin): t is Plugin {
 /**
  * Utility function to enable a plugin.
  */
-export function enableIf<PluginContextType = {}>(
+export function enableIf<PluginContextType extends Record<any, any> = {}>(
```

Comment:
technically this is a breaking change 🤔

Reply:
but could also be considered as a bug fix, as the earlier type did not satisfy the usage constraint...